ruby/mediaflux.rb

```diff
@@ -0,0 +1,133 @@
+require 'net/http'
+require 'rexml/document'
+
+module MediaFlux
+
+  class MFError < StandardError
+  end
+
+  class MFApiError < MFError
+    attr_reader :error, :message, :stack
+    def initialize(response_doc)
+      reply = response_doc.elements['/response/reply']
+      @error = reply.elements['error'].text
+      @message = reply.elements['message'].text
+      @stack = reply.elements['stack'].text
+    end
+  end
+
+  class MFXmlError < MFError
+  end
+
+  class MFClient
+    def initialize(
+        readable: false, verbose: false,
```

Comment:
@mccalluc I a bit confused by this readable parameter.  Is the intention to stop utilizing method missing?

Reply:
I introduced `readable` after finishing `demo_1`... So I introduced it with a default of `false` out of a misguided sense of backwards-compatibility.

If folks are sold on this approach, happy to make it the default.